src/plugins/discover/public/embeddable/search_embeddable_component.tsx

```diff
@@ -51,8 +52,10 @@ export function SearchEmbeddableComponent({ searchProps }: SearchEmbeddableProps
         data-test-subj="embeddedSavedSearchDocTable"
       >
         {discoverEmbeddableProps.totalHitCount !== 0 ? (
-          <EuiFlexItem>
-            <DataGridTableMemoized {...discoverEmbeddableProps} />
+          <EuiFlexItem style={{ minHeight: 0 }}>
+            <div className="osdDocTable__container">
```

Comment:
Why not just give the class name to the flex item?

Reply:
Good call. I thought EuiFlexItem is using `displayName`. Apparently, I remembered this wrong. I fixed the comment.